pkg/generators/models/testdata/cases/oneof/expected/model_bar.go

```diff
@@ -5,5 +5,37 @@
 //     Version: 0.1.0
 package generatortest
 
-// Bar is a value type.
-type Bar interface{}
+import (
```

Comment:
I think this would be a breaking change,  so i updated the title of the PR to be `feat!:`